content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,23 +1,33 @@
 ---
-title: 'Deploy to AWS Lambda with Serverless Framework'
-metaTitle: 'Deploy your Serverless Framework project with Prisma'
-metaDescription: 'Learn how to deploy your Prisma-backed applications to AWS Lambda with Serverless Framework'
+title: 'Deploy to AWS Lambda'
+metaTitle: 'Deploy your Prisma-backed project to AWS Lambda'
+metaDescription: 'Learn how to deploy your Prisma-backed applications to AWS Lambda with AWS SAM, Serverless Framework, or SST'
 tocDepth: 3
 ---
 
 <TopBlock>
 
-This guide explains how to avoid common issues when deploying a project using Prisma with [Serverless Framework](https://www.serverless.com/framework/).
+This guide explains how to avoid common issues when deploying a project using Prisma to [AWS Lambda](https://aws.amazon.com/lambda/).
 
-The Serverless Framework simplifies deployment to AWS Lambda. It provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
+While a framework is not required to deploy to AWS Lambda, this guide covers deploying with [AWS SAM](https://aws.amazon.com/serverless/sam/), [Serverless Framework](https://www.serverless.com/framework/), and [SST](https://sst.dev/).
```

Comment:
We can use a bullet list here.
```suggestion
While a framework is not required to deploy to AWS Lambda, this guide covers deploying with: 
- [AWS SAM](https://aws.amazon.com/serverless/sam/)
- [Serverless Framework](https://www.serverless.com/framework/)
-  [SST](https://sst.dev/).
```

Follow-up questions: 
- Do we want to have the Serverless Framework & SST guides in a separate page or the same page? I think we should have them in separate pages to keep the guides concise. I can see there's some content on them on this page. 
- If it's possible to deploy to AWS without a framework, should we show that first and then create separate guides for the different frameworks? 🤔

Reply:
Put this in the PR description but we discuss it more:
```
I thought of the following three choices:

1. A mega page with both generic tips and framework-specific tips
2. A page for generic tips and then subpages for framework-specific tips
3. Pages for framework-specific tips with a "generic" section at top of each.

I chose 1 as it did not repeat content (which could become out of sync) and made it so that all steps a user should follow are on the same page.
```
@janpio @nikolasburk do either of you have opinions on structure here?

Reply:
No opinion - as long as we have the content.
We can start with 0 page, and split out later.

But if one can deploy without any of these, that should indeed be the first option explained here.